lxd/containers_post.go

```diff
@@ -43,6 +43,8 @@ type containerImageSource struct {
 
 	/* for "copy" type */
 	Source string `json:"source"`
+	/* for "migration" type. Whether the migration is live. */
+	Live bool `json:"live"`
```

Comment:
When pushing, the secrets aren't passed as both source and target are expected to generate and return a set of websocket, but not actually connect to each another.

@brauner actually was passing secrets there before exactly as you suggested, but it seemed a bit confusing to send information which is effectively secret and not needed by that party at all (the client is the one who needs all the secrets).